components/animated/newsFeedLoaders/ContainerLoader.tsx

```diff
@@ -0,0 +1,29 @@
+import { StyleSheet, View } from 'react-native';
+import React from 'react';
+import SkeletonCard from './OneFeedLoader';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    paddingTop: 10,
+    textAlign: 'left',
+    width: '100%',
+    paddingHorizontal: 8,
+
+    backgroundColor: '#15192C',
+  },
+});
+
+export default function ContainerLoader() {
+  return (
+    <View style={styles.container}>
+      <SkeletonCard />
+      <SkeletonCard />
+      <SkeletonCard />
+      <SkeletonCard />
+      <SkeletonCard />
+      <SkeletonCard />
+      <SkeletonCard />
```

Comment:
![ah-shit-here-we-go-again-ah-shit](https://user-images.githubusercontent.com/19322697/170226731-b16bfeba-1028-4450-9278-2e80a36aa046.gif)